### PR TITLE
Quick fix, to make the conditional termination module actually work...

### PR DESCRIPTION
**Pull request by DarkMatter222**
…for at least the base cases (conditional-termination.t2 and conditional-termination7.t2)

### CODE REVIEW

File: src/T2-CLI.fsproj [hunk 1]
```diff
@@ -101,9 +101,28 @@
     <Reference Include="FSharp.Core, Version=$(TargetFSharpCoreVersion), Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
       <Private>True</Private>
     </Reference>
+    <Reference Include="FSharp.PowerPack">
```
Comment (mmjb) [inline src/T2-CLI.fsproj]: Why do you need these?

File: src/conditional.fs [hunk 2]
```diff
@@ -330,7 +333,7 @@ let pre_synth (rho:formula, econd:formula, vars) =
 /// with phase change. 
 let rec pre_synth_phase (rho:formula, vars, cur_constr:formula, econd, depth_left) = 
 
-
+    printfn "HELLO!"
```
Comment (mmjb) [inline src/conditional.fs]: *cough*

Comment (DarkMatter222) [inline src/conditional.fs]: Sorry, that's something automatically generated after me trying to run T2
for ages. I probably don't need it. Sorry, I'm doing three things at the
same time and I'm kind of overlooking all of them. Let mi fix this.

Comment (DarkMatter222) [inline src/conditional.fs]: I can start coughing too, though - are there any other changes I made to
files different form conditional.fs that you didn't commit, apart of the
one in formula.fs?

**Pull request closed without merging**